code/game/machinery/computer/arcade.dm

```diff
@@ -336,10 +336,17 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	enemy_name = "Cuban Pete"
 	name = "Outbomb Cuban Pete"
 
-
 	updateUsrDialog()
 
-
+/obj/machinery/computer/arcade/battle/on_deconstruction()
+	. = ..()
+	if((obj_flags & EMAGGED))
+		if(ismob(usr))
+			to_chat(usr, "<span class='userdanger'>Trying to cheat me? Chick-chicky-boom, chick-chicky boom!</span>")
+			usr.gib()
```

Comment:
gibbing is too much maybe remove their legs or something

Reply:
This is how it works right now if you lose the arcade.